NEWS.md

```diff
@@ -14,6 +14,8 @@
   for the degrees of freedom, another argument `vcov` is added to specify the
   method used to adjust the coefficients covariance matrix.
 - Add optional `tidymodels` framework support.
+- Add Satterthwaite degrees of freedom support for empirical and empirical Jackknife
```

Comment:
Maybe merge this with the bullet in line 12 ff.? since empirical cov matrix is only added in the new release

Reply:
Or leave it separate but then also add the residual d.f. news (I don't see that yet)